### PR TITLE
DROOLS-2436: Fix wrong diagram's name match condition.

### PR DESCRIPTION
**Pull request by romartin**
Hey @manstis  @jomarko 

Fixed the issue on the diagram's name matching condition. 

See [DROOLS-2436](https://issues.jboss.org/browse/DROOLS-2436)

Sorry it was my fault, added a small test at least for this. 

@manstis Thanks for pointing out the root issue, this way things are easier... :+1:  

Thanks!

### CODE REVIEW

**Comment by romartin** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
@manstis  hahah no no... the `9` was due to my finger at the wrong location... :)

File: kie-wb-common-stunner/kie-wb-common-stunner-backend/src/main/java/org/kie/workbench/common/stunner/backend/service/DiagramLookupServiceImpl.java [hunk 1]
```diff
@@ -83,7 +83,7 @@ protected boolean matches(final String criteria,
                               final Diagram<Graph, Metadata> item) {
         final Map<String, String> criteriaMap = AbstractCriteriaLookupManager.parseCriteria(criteria);
         final String name = criteriaMap.get(DiagramLookupRequest.CRITERIA_NAME);
-        if (null != name && name.trim().length() > 9) {
+        if (null != name && name.trim().length() > 0) {
             return name.equals(item.getName());
```
Comment (jomarko) [inline kie-wb-common-stunner/kie-wb-common-stunner-backend/src/main/java/org/kie/workbench/common/stunner/backend/service/DiagramLookupServiceImpl.java]: Just question due to if condition, shouldn't we return `name.trim().equals(item.getName())` ?

Comment (romartin) [reply kie-wb-common-stunner/kie-wb-common-stunner-backend/src/main/java/org/kie/workbench/common/stunner/backend/service/DiagramLookupServiceImpl.java]: well don't think so, or at least, in that case you'll have to do trim() as well on the item. So imagine you have a diagram called "my diagram", if you do the trim that will make the equals return false..

**Pull request merged**